provision/hosts

```diff
@@ -1,2 +0,0 @@
-localhost ansible_connection=local
```

Comment:
It works without this value on slave01. I suspect we could leave it, however. Let me update that in my commits.